nodejs/eks/transform.ts

```diff
@@ -44,6 +44,8 @@ class Transform<T, U> extends dynamic.Resource {
  * transform evaluates the given function on the given input iff the input's value has changed and returns the result
  * as an output.
  */
+// `transform` is no longer used internally in the EKS library. We should consider marking this function as @deprecated
+// and removing it from this library in the future.
 export default function transform<T, U>(name: string, input: T, func: (v: T) => U, opts?: pulumi.CustomResourceOptions): pulumi.Output<U> {
```

Comment:
Seems best to go ahead and mark this as `@deprecated` but wanted to see what folks thought first.

Reply:
Probably fine to deprecate. I'd be surprised if it was used by anything else.